src/parser/SHOGunApplicationUtil.ts

```diff
@@ -266,7 +269,10 @@ class SHOGunApplicationUtil<T extends Application, S extends Layer> {
       projection,
       params: {
         'LAYERS': layerNames,
-        'TRANSPARENT': transparent !== undefined ? transparent : true
+        'TRANSPARENT':
+          requestParams?.transparent !== undefined
+            ? requestParams?.transparent
+            : true
```

Comment:
This doesn't take advantage of the `requestParams` object yet, I would favor an implementation like follows:

```ts
params: {
  'LAYERS': layerNames,
  {...requestParams}
}
```